server/models/subscription.ts

```diff
@@ -12,6 +17,7 @@ module.exports = (sequelize, DataTypes) => {
     category_id: { type: DataTypes.STRING, allowNull: false },
     object_id: { type: DataTypes.STRING, allowNull: false },
     is_active: { type: DataTypes.BOOLEAN, defaultValue: true, allowNull: false },
+    immediate_email: { type: DataTypes.BOOLEAN, defaultValue: false, allowNull: false },
```

Comment:
This seems a bit weird to directly have this boolean here, even in the short term. We'll be extending this with the other options (daily, weekly, monthly, etc etc), might as well make the change now